### ISSUE TITLE
Cookie problem? (auth)

### ISSUE THREAD

**Issue by TaciteOFF** [openfoodfacts/openfoodfacts-server#473]
Hello.

1. Disconnect from fr.openfoodfacts.org and world.openfoodfacts.org
2. Login on fr.openfoodfacts.org
3. Go to world.openfoodfacts.org (you don't appear to be logged in)
4. Go back to fr.openfoodfacts.org

You're no longer logged in

**Comment by hangy** [comment]
Interestingly not reproducible from de.openfoodfacts.org (and a few other country subdomains) or world-fr.openfoodfacts.org. Only when switching to fr.openfoodfacts.org, the user is logged out. The session cookie is actually sent by the client, but is cleared by the server in the response.
I was not able to reproduce the error on my own development instance of Product Opener. It looks like maybe they're not even on the same server, world.openfoodfacts.org resolves to a different IP than fr.openfoodfacts.org.

**Comment by stephanegigandet** [comment]
Thanks for investigating @hangy, I had changed the *.openfoodfacts.org to the new server  ip address, but there was also a fr.openfoodfacts.org record that I didn't remember creating. I removed the record.

So fr.openfoodfacts.org queries were proxied from the old server to the new, and I wasn't able to get the right user ip, so that explains the disconnects.

Should be fixed soon when DNS caches expire.
